kpi/models/asset.py

```diff
@@ -302,7 +302,20 @@ def _prioritize_translation(self, content, translation_name, is_new=False):
         if is_new and (translation_name in _translations):
             raise ValueError('cannot add existing translation')
         elif (not is_new) and (translation_name not in _translations):
-            raise ValueError('translation cannot be found')
+            # if there are no translations available, don't try to prioritize,
+            # just ignore the translation `translation_name`
+            if len(_translations) == 1 and _translations[0] is None:
+                return
+            else:  # Otherwise raise an error.
+                # Remove None from translations we want to display to users
+                valid_translations = [t for t in _translations if t is not None]
+                raise ValueError("Translation `{translation_name}` could not be found in your form. "
```

Comment:
I think we should tell the user specifically that their default language setting is the problem. Can we move this logic to `_make_default_translation_first()`? I realize `_translations = content['translations']` or similar will have to be duplicated. Then, we could make the error message something like:
```
`{translation_name}` is specified as the default language, but only these translations are present in the form: `{translations}`.
```
I like the "Please select..." language, but let's keep the message concise at the expense of courtesy :)

Reply:
Done!